tests/tests-mayastor/Cargo.toml

```diff
@@ -20,8 +20,8 @@ opentelemetry-jaeger = { version = "0.15.0", features = ["tokio"] }
 tracing-opentelemetry = "0.15.0"
 opentelemetry = "0.16.0"
 actix-web-opentelemetry = "0.11.0-beta.5"
-tracing = "0.1.26"
-anyhow = "1.0.43"
+tracing = "0.1.28"
+anyhow = "1.0.44"
```

Comment:
btw now you don't have to revert changes to the Cargo.toml from the openapi, it should work (well, I hope so).

Reply:
Oh okay, I did do that, I can try it out, would be nice to be able to upgrade it everywhere in one blow!